src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -93,7 +93,18 @@ message SymbolizeTaskOutput {
 }
 
 message AnalyzeTaskOutput {
- // TODO(metzman): Fill this out.
+  optional int64 crash_revision = 1;
```

Comment:
Are we sure this is everything to save?

Reply:
To create this list, I looked at all the testcase fields updated within utask_main. This is normally everything (unless I missed some field, hopefully not)

Reply:
Did you look in setup.py as well?

Reply:
Yes, I also checked setup.py, and as far as I saw, it only reads data from testcase and never updates it.